windows/CodePushReactPackage.cs

```diff
@@ -1,25 +1,18 @@
 ﻿using Newtonsoft.Json.Linq;
-using Newtonsoft.Json;
 using ReactNative;
 using ReactNative.Bridge;
 using ReactNative.Modules.Core;
 using ReactNative.UIManager;
 using System;
 using System.Collections.Generic;
-using System.Reflection;
 using System.Threading.Tasks;
-using Windows.ApplicationModel;
-using Windows.Storage;
-using Windows.Web.Http;
+
 
 namespace CodePush.ReactNative
 {
     public sealed class CodePushReactPackage : IReactPackage
     {
         private static CodePushReactPackage CurrentInstance;
-        private static bool NeedToReportRollback = false;
-
```

Comment:
This is suspicious, is this some missing feature, or was it just totally dead code?